### DISCUSSION TITLE
Rip VIDEO DVDs to ISO

### DISCUSSION THREAD

**Discussion by drlauridsen** [automatic-ripping-machine/automatic-ripping-machine#870] [Q&A]
I am very impressed with ARM, still trying to figure out a couple of details, including how to include subtitles and getting notifications with Gotify working.

However before I discovered ARM, I was actually just looking for an automated way of converting my DVDs to ISO files, since KODI can play ISOs..

Although it obviously defies a lot of the features with ARM - is it possible to setup ARM to simply rip video DVDs to ISOs - maintaining the lookup in movie databases and naming the files.

Thanks for the impressive work

**Comment by microtechno9000** [comment]
ARM doesn't copy disk meta data when making an iso, that is, look for movie titles.

Work is going into allowing for more flexibility with sessions to modify the process. However, with only a few devs it takes time. If you can code, or able to test. Please help out
